pkg/release/server.go

```diff
@@ -30,13 +31,23 @@ func (s *Server) SummarizeReleases(ctx context.Context, req *proto.ListReleasesR
 		return nil, err
 	}
 	listReq := rls.ListReleasesRequest{
```

Comment:
Can we do this based on user input like Helm cli?

Reply:
A query param all=true/false can be added.